### PR TITLE
Add Mvc.Api.Analyzers to 2.2.6 patch

### PR DESCRIPTION
**Pull request by pranavkm**

### CODE REVIEW

**Comment by pranavkm** [comment]
I wasn't super what the values for ZipManifestGenerator are. Is this package included as part of the package archive?

File: version.props [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <PropertyGroup>
     <AspNetCoreMajorVersion>2</AspNetCoreMajorVersion>
     <AspNetCoreMinorVersion>2</AspNetCoreMinorVersion>
-    <AspNetCorePatchVersion>5</AspNetCorePatchVersion>
+    <AspNetCorePatchVersion>6</AspNetCorePatchVersion>
```
Comment (natemcmaster) [inline version.props]: Would recommend reverting this for the purpose of this PR. There are several steps required to update the repo to build a new patch (steps documented in https://github.com/aspnet/AspNetCore/blob/release/2.2/docs/PreparingPatchUpdates.md).

https://github.com/aspnet/AspNetCore-Internal/issues/2593 tracks completing these steps for 2.2.6 and 2.1.12, and should be done in the next week or so.

**Comment by pranavkm** [comment]
🆙 📅

**Review by natemcmaster — APPROVED** [review]

**Comment by pranavkm** [comment]
cc @dougbu

**Pull request merged**